### ISSUE TITLE
Xbox One controller button activates multiple times, but only when two controllers are connected?

### ISSUE THREAD

**Issue by nyarsulik** [Microsoft/xbox-live-unity-plugin#260]
Unity Bug Report:
https://fogbugz.unity3d.com/default.asp?1099111_n1n7qlp3auqm00hi

I am not sure if this is a Unity issue or not, but my games (that have been in the store for some time now) randomly started having an issue when multiple controllers are connected.  UI movement works fine with one controller, but the second you add a second controller pressing the A button (which is tied to the "Submit" function in Unity) triggers rapidly, essentially breaking the game when multiple controllers are active.  With this issue, one of my users can't select anything in the menu (it automatically chooses level 1 since it triggers so quickly).

Has anyone else seen this issue?

This does not happen in Unity editor, but happens when running within Visual Studio and also on the Xbox One console itself.

**Comment by madcowie** [comment]
I also have this problem. Sad nobody has got back to you :( Did you find a solution?

**Comment by nyarsulik** [comment]
On January 25th Unity got back to me but I didn't see anything further.  They were able to reproduce it on their side so he said he would try to get someone working on it.  I honestly don't know when the expected fix date is.  Is it really the same for you, only when two controllers are being used?  It is such a strange problem.

**Comment by madcowie** [comment]
Yes mate exactly the same for me using unity 2018.2 on this project. I also have 2018.3 but decided I didn't wanna break this project as it's 80% done 😂. You have this issue on .3 and it's the same? I Thought it was me just being daft 😁
